java/server/src/org/openqa/selenium/grid/graphql/selenium-grid-schema.graphqls

```diff
@@ -23,7 +24,9 @@ type GridQuery {
 }
 
 type Grid {
-  url: Url!
+  uri: Uri!
+  host: String!
```

Comment:
The URI contains both the host and the port. Please don't duplicate the information